sporkfish/searcher/minimax.py

```diff
@@ -59,34 +62,64 @@ def __init__(
             logging.info("Disabled transposition table in search.")
 
         self._evaluator = evaluator
-        self._move_order = move_order
+
+        # Killer move table - storing quiet beta-cut off moves
```

Comment:
[Minor/idea] I would prefer to, either:

- have an if statement here - only initialise/declare this kill move table if the config is on, or
- we move this into a different class (with all the move ordering methods and attributes defined there), and have `MinimaxVariant` inherits from it? 
 
Again, I am not too fussed about it, can always refactor later. Just that my concern is `MinimaxVariant` and `negamax` are getting very big.

[edit] re point number 2 - sorry just noticed your reply on how to categorise TT-related move orders. Continuing that discussion below.

Reply:
Marking resolved, continued below